gpu/segmentation/include/pcl/gpu/segmentation/impl/gpu_extract_clusters.hpp

```diff
@@ -100,80 +115,59 @@ pcl::gpu::extractEuclideanClusters (const typename pcl::PointCloud<PointT>::Ptr
     // once the area stop growing, stop also iterating.
     do
     {
-      // Host buffer for results
-      std::vector<int> sizes, data;
-
+      data.clear();
       // if the number of queries is not high enough implement search on Host here
       if(queries_host.size () <= 10) ///@todo: adjust this to a variable number settable with method
       {
-        std::cout << " CPU: ";
+        PCL_DEBUG(" CPU: ");
+        //data.clear();
```

Comment:
remove this commented code?

Reply:
Oh, yes! This line shouldn't be there. Thank you!